tests/integration/delete.bats

```diff
@@ -47,6 +47,52 @@ function teardown() {
   [ "$status" -ne 0 ]
 }
 
+@test "runc delete --force with subcgroups in v2" {
+  requires cgroups_v2
+  [[ "$ROOTLESS" -ne 0 ]] && requires rootless_cgroup
+  set_cgroups_path "$BUSYBOX_BUNDLE"
+
+  # grant `rw` priviledge to `/sys/fs/cgroup`
+  DATA=$(cat <<EOF
+    "destination": "/sys/fs/cgroup",
+    "type": "cgroup",
+    "source": "cgroup",
+    "options": [
+            "nosuid",
+            "noexec",
+            "nodev",
+            "relatime",
+            "rw"
+    ]
+EOF
+    )
+  DATA=$(echo ${DATA} | sed 's/\n/\\n/g' | sed 's/\//\\\//g')
+  sed ":a;N;s/\n//g;ta" config.json | sed "s/\"destination\": \"\/sys\/fs\/cgroup\",[^}]*/${DATA}/" > ${BUSYBOX_BUNDLE}/config.json
+ 
+  # run busybox detached
+  runc run -d --console-socket $CONSOLE_SOCKET test_busybox
+  [ "$status" -eq 0 ]
+
+  # check state
+  testcontainer test_busybox running
+
+  # create subcgroups
+  runc exec -t test_busybox mkdir -p /sys/fs/cgroup/foo/bar
```

Comment:
How about change this line to `mkdir -p $CGROUP_PATH/foo/bar`?
I think it is enough for this test case.